index.bs

```diff
@@ -786,9 +813,24 @@ partial interface HTMLIFrameElement {
       resulted only in this report being generated (with no further action taken
       by the user agent in response to the violation).
 
-      Note: There is currently no mechanism in place for enabling report-only
-      mode, so [=PermissionsPolicyViolationReportBody/disposition=] will always
-      be set to "enforce".
+  <section>
+    <h3 id="permissions-policy-report-only-http-header-field">`Permissions-Policy-Report-Only`
+    HTTP Header Field</h3>
+    <p>The &#96;<dfn export http-header
+    id="permissions-policy-report-only-header"><code>Permissions-Policy-Report-Only</code></dfn>&#96;
+    HTTP header field can be used in the [=response=] (server to client) to
+    communicate a <a>permissions policy</a> that should not be enforced by the
+    client, but instead should be used to trigger reports to be sent if any
+    policy declared within it *would* have been violated, had the policy been
+    active.</p>
+    <p><a http-header>Permissions-Policy-Report-Only</a> is a structured header.
```

Comment:
nit: My recommendation for all references to headers, (like my above recommendation) would be to wrap these in escaped backticks (or the literal `&#96;`) to get the code formatting. I think that's pretty common in specs, and makes it more clear